contracts/script/eigen/DeployEigenLayer.s.sol

```diff
@@ -35,6 +32,11 @@ contract DeployLocalEigenLayer is Script, EigenLayerLocal {
         string memory strategiesJson;
         for (uint256 i = 0; i < deps.strategies.length; i++) {
             IStrategy strat = IStrategy(deps.strategies[i]);
+
+            if (address(strat) == beaconChainETHStrategy) {
+                continue;
+            }
+
```

Comment:
`make deploy-eigen-save-anvil-state` was broken

Reply:
Should we add a TODO to revert this later?

Reply:
No this fixed it